files/default/neutron-ha-tool.py

```diff
@@ -233,6 +236,10 @@ def run(args):
     else:
         raise ValueError('Invalid agent_selection_mode')
 
+    if args.target_agent is not None:
+        Configuration.agent_picker_class = SingleAgentPicker
+        SingleAgentPicker.agent_selection_value = args.target_agent
```

Comment:
Yes, I feel your pain. Modifying class variables is sub-optimal. What we could do is to change what is being stored in `agent_picker_class`, and let's say we store an `agent_picker` - which will be an instance. In that case we would need to add one further parameter to the agent picker interface, something like `set_agents`. Let me add a commit, and see if that design would make it any better.